optifit backend/app.py

```
@@ -37,108 +60,326 @@ def error_response(message, status_code):
+    print(f"   Validation: {'✅ Available' if VALIDATION_AVAILABLE else '❌ Basic validation'}")
+    print("📊 Endpoints:")
+    print("   GET  / - Server info")
+    print("   GET  /ping - Health check")
+    print("   GET  /health - Detailed health status")
+    print("   POST /upload - Upload exercise video")
+    print("   GET  /result/<job_id> - Get analysis results")
+    print("   GET  /jobs - List all jobs")
+    
     # Start Flask app
-    app.run(host="0.0.0.0", port=5000)
+    app.run(host="0.0.0.0", port=5000, debug=True)
```

Ruff (0.14.1)

Possible binding to all interfaces

(S104)